modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Tests/TestData/Sources/ScriptBoilerplate.cs

```diff
@@ -0,0 +1,33 @@
+using Godot;
+
+public partial class ScriptBoilerplate : Node
+{
+    private NodePath _nodePath;
+    private int _velocity;
+
+    public override void _Process(double delta)
+    {
+        _ = delta;
+
+        base._Process(delta);
+    }
+
+    public int Bazz(StringName name)
+    {
+        _ = name;
+        return 1;
+    }
+
+    public void IgnoreThisMethodWithByRefParams(ref int a)
+    {
+        _ = a;
+    }
+}
+
+partial struct OuterClass
+{
+    public partial class NesterClass : RefCounted
```

Comment:
Good catch lol, it's probably been here for a while 😂 https://github.com/godotengine/godot/blob/master/modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Sample/ScriptBoilerplate.cs#L31

Reply:
Oh I see, now that I think about it maybe it was meant to be called `Inner` since the containing class is called `Outer`. But I kinda like `Nested`. It's not important anyway.